stoplight.js

```diff
@@ -0,0 +1,13 @@
+alert ("working")
+
+$(document).ready(function () {
+    ('red').click(function(e) {
+
+        ('stoplight').removeClass('active');
+
+        var $this = $(this);
+        if (!$this.hasClass('active')) {
+            $this.addClass('active');
+        }
+    });
+});
```

Comment:
You're on the right track with some of this code but you need to make you can select things properly before moving on to the logic. Tackle these problems in small steps.